payload/platformsh/.platform.app-2x.yaml

```diff
@@ -80,9 +82,23 @@ runtime:
         - readline
         - msgpack
         - redis
+        #- apcu
+        #- name: 'blackfire'
+        #  configuration:
+        #      server_id: 'xxxx-xxx-xxx-xxx-xxxx'
+        #      server_token: 'xxxx'
+
 
 variables:
     php:
         memory_limit: 256M
+        # framework -> session -> handler_id => in your config set to null will use default session handler from php.ini
         session.save_handler: redis
-        session.save_path: "tcp://cache.internal:6379"
+        session.save_path: "tcp://sessions.internal:6379"
```

Comment:
We don't have this in upsteam. @Crell strongly suggested it can cause issues setting it like this.

Reply:
Hard coding this value here makes it impossible to use the same code base on both PS and PE.  You don't want that. :-)